modules/standard/DateTime.chpl

```diff
@@ -1368,6 +1368,20 @@ module DateTime {
     }
   }
 
+  pragma "no doc"
+  proc -(dt: datetime, t: time):timedelta {
```

Comment:
I don't feel like this operator makes a whole lot of sense. It might make more sense to me if the `castTime` used a date of 0 instead, but I still don't know when I'd want to use this. I don't think we should add this version.

Reply:
I thought about cases when we probably need to adjust the time of particular datetime as for the date 0 part we already have the ```timeSinceEpoch``` procedure under review.

Reply:
@daviditen got your point if am correct as the user is aware of ```time``` difference he probably won't need to convert it to ```timedelta``` which is basically what this operator does. I will remove this, although unrelated would it be nice if we can have it return a new ```datetime``` object which subtracts time as provided.